### ISSUE TITLE
Incomplete alias check

### ISSUE THREAD

**Issue by ikhvostenkov** [yegor256/netbout#627]
Check here:
```Java
public String check(final String name) {
        final String answer;
        if (name.length() < Tv.FOUR) {
            answer = "too short, must be 4 letters at least";
        } else if (name.length() > Tv.TWENTY) {
            answer = "too long, must be 20 letters at most";
        } else if (name.matches("[a-z0-9]+")) {
            if (new Everybody(this.region).occupied(name)) {
                answer = "this alias is occupied";
            } else {
                answer = "";
            }
        } else {
            answer = "only English letters and numbers are accepted";
        }
        return answer;
    }
```
According with code above, if you enter alias "Test", you will get error message "only English letters and numbers are accepted", English letters are used in alias, but in upper case. Lets change current notification or add one more check with separate error message.

**Comment by dmzaytsev** [comment]
@ikhvostenkov you're right, thanks

**Comment by dmzaytsev** [comment]
@karato it's a valid bug

**Comment by dmzaytsev** [comment]
@karato please set milestone to 3.0

**Comment by dmzaytsev** [comment]
@ikhvostenkov I believe we have to extend the regexp above to allow use mixed case for an alias.
Because Github allows to use characters in the upper and lower case  in username. The netbout accounts generally use GitHub auth.

**Comment by ikhvostenkov** [comment]
@dmzaytsev This will be even better to allow upper case.

**Comment by yegor256** [comment]
@ikhvostenkov @dmzaytsev this pattern is correct: `[a-z0-9]+`. We don't want upper case letters in aliases.

**Comment by karato** [comment]
> @karato it's a valid bug

@dmzaytsev tag "bug" added

**Comment by karato** [comment]
@ikhvostenkov milestone set to `3.1` (correct me if I am wrong)

**Comment by karato** [comment]
> @karato please set milestone to 3.0

@dmzaytsev sure, milestone set to 3.0

**Comment by karato** [comment]
@ikhvostenkov many thanks, **15 mins** added to your acc for reporting this bug, pmt ID `58040546`

**Comment by dmzaytsev** [comment]
@karato this is postponed task

**Comment by karato** [comment]
> @karato this is postponed task

@dmzaytsev right, I added "postponed" tag

**Comment by karato** [comment]
> @karato this is postponed task

@dmzaytsev right, I will find someone else, no problem

**Comment by dmzaytsev** [comment]
@ikhvostenkov fixed in  #657 please close

**Comment by ikhvostenkov** [comment]
@dmzaytsev Verified - Fixed.
